test/forge/BlueTest.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../src/adaptive-curve-irm/interfaces/IAdaptiveCurveIrm.sol";
+import "../../src/fixed-rate-irm/interfaces/IFixedRateIrm.sol";
+
+import {IMorpho, MarketParams} from "../../lib/morpho-blue/src/interfaces/IMorpho.sol";
+import {MathLib} from "../../lib/morpho-blue/src/libraries/MathLib.sol";
+import {MarketParamsLib} from "../../lib/morpho-blue/src/libraries/MarketParamsLib.sol";
+import {ORACLE_PRICE_SCALE} from "../../lib/morpho-blue/src/libraries/ConstantsLib.sol";
+
+import "../../lib/forge-std/src/Test.sol";
+import {ERC20Mock} from "../../lib/morpho-blue/src/mocks/ERC20Mock.sol";
+import {OracleMock} from "../../lib/morpho-blue/src/mocks/OracleMock.sol";
+
+contract BlueTest is Test {
+    using MathLib for uint256;
+    using MarketParamsLib for MarketParams;
+
+    event BorrowRateUpdate(Id indexed id, uint256 avgBorrowRate, uint256 rateAtTarget);
+
+    uint256 internal constant MIN_TEST_AMOUNT = 100;
+    uint256 internal constant MAX_TEST_AMOUNT = 1e28;
+    uint256 internal constant MIN_TIME_ELAPSED = 10;
+    uint256 internal constant MAX_TIME_ELAPSED = 315360000;
```

Comment:
where dois this value come from?

Reply:
for `MAX_TEST_AMOUNT` and `MIN_TEST_AMOUNT` I used the same ones as we use for blue tests
`MIN_TIME_ELAPSED` is 10sec (approx one block)
`MAX_TIME_ELAPSED` is 315360000 sec (ten years)

Reply:
i think we can just use `10 year` instead then